src/cluster.c

```diff
@@ -4537,7 +4540,7 @@ NULL
 
         if (c->argc == 5) {
             if (getLongLongFromObject(c->argv[4], &cport) != C_OK) {
-                addReplyErrorFormat(c,"Invalid TCP bus port specified: %s",
+                addReplyErrorFormat(c,"Invalid TCP bus port or cluster port specified: %s",
```

Comment:
I believe the TCP bus port is intended to be the cluster port, so I don't think this addition is needed. You can keep the reword if you think it's clearer.